spel/minimal-linux.json

```diff
@@ -212,7 +212,7 @@
             "name": "minimal-rhel-7.3-hvm",
             "type": "amazon-ebs",
             "ami_name": "{{ user `spel_identifier` }}-{{ build_name }}-{{ user `spel_version` }}.x86_64-gp2",
-            "ami_description": "This is an STIG-partitioned [**NOT HARDENED**], LVM-enabled, \"minimal\" RHEL 7.3 AMI, with \"billingProduct\" set for proper billing and licensed access to RedHat yum repos. This AMI contains updates through {{ isotime \"2006-01-02\" }}. For details, see {{ user `desc_url` }}.",
+            "ami_description": "This is a STIG-partitioned [**NOT HARDENED**], LVM-enabled, \"minimal\" RHEL 7.3 AMI (yum and license chargeback included) with updates through {{ isotime \"2006-01-02\" }}. See {{ user `desc_url` }} for details.",
```

Comment:
Normalizing the var-name and description fields (sorry, misread your earlier: didn't realize you were referring to 6 and it hadn't length-failed because I hadn't run it).

Doing test-run, now.